src/main/java/com/linkedin/kmf/apps/SingleClusterMonitor.java

```diff
@@ -245,7 +254,7 @@ private static ArgumentParser argParser() {
   public static void main(String[] args) throws Exception {
     ArgumentParser parser = argParser();
     if (args.length == 0) {
-      System.out.println(parser.formatHelp());
+      LOG.info(parser.formatHelp());
```

Comment:
This is command line help and needs to be reported to the user on standard output or standard error.